### ISSUE TITLE
Some details

### ISSUE THREAD

**Issue by nice2-cu** [SanaNazari/LMFLoss#2]
I would like to know some details like the choice of hyperparameters and the choice of parameters in the loss function. How to get the `weight` in the loss function? Thanks
